classes/components/forms/FieldContrastColorPicker.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * @file classes/components/form/FieldContrastColorPicker.php
+ *
+ * Copyright (c) 2014-2025 Simon Fraser University
+ * Copyright (c) 2000-2025 John Willinsky
```

Comment:
New files are just receiving the starting year

```suggestion
 * Copyright (c) 2025 Simon Fraser University
 * Copyright (c) 2025 John Willinsky
```